README.md

```diff
@@ -100,61 +115,91 @@ List of references used for each project on the Portal with `CellAssign`, includ
 
-The `figures` and `tables` folders contain the most up-to-date version of each of the figures and tables, respectively.
-The `scripts` directory contains all scripts used to create the figures and tables.
-See the [`README` for the `scripts` directory](./scripts/README.md) for more information on figure and table scripts.
 
-To generate all figures and tables, run the script [`generate-figures-tables.sh`](generate-figures-tables.sh) as:
+The [`generate-figures-tables.sh`](generate-figures-tables.sh) script can be used to prepare all figures and tables.
+The script assumes that a folder called `s3_files` has been populated with relevant data files.
+Therefore, **before running this script** you will first need to prepare these input data files, as described in the sections below.
+
+Then, run the figure generation script as:
 
 ```sh
 bash generate-figures-tables.sh
 ```
 
-Note that this script assumes that the `s3_files` directory has been populated with relevant data files.
-These files can be obtained by first running the figure setup scripts, which currently require S3 bucket access as a Data Lab member.
-Setup scripts can be run as:
+### Instructions to prepare data
+
+If you are not a member of the Data Lab, please follow the instructions provided in `reproduce-figures-analysis/obtain-prepare-data.md` to obtain and prepare additional data needed to regenerate figures and tables.
+
+### Instructons for Data Lab members
+
+To prepare data for figure and table generation, will need to run the figure setup scripts:
 
 ```sh
 Rscript scripts/figure_setup/sync-metadata.R
 Rscript scripts/figure_setup/sync-data-files.R
 Rscript scripts/figure_setup/sync-reference-files.R
+Rscript scripts/figure_setup/assign-brain-classifications.R
+Rscript scripts/figure_setup/prepare-sample-whitelist.R
 ```
 
-If you have setup `1Password` to handle your AWS credentials, you will need to prefix those lines with `op run --`:
+If you have setup `1Password` to handle your AWS credentials, you will need to prefix scripts beginning with `sync-` with `op run --`, specifically:
 
 ```sh
 op run -- Rscript scripts/figure_setup/sync-metadata.R
 op run -- Rscript scripts/figure_setup/sync-data-files.R
 op run -- Rscript scripts/figure_setup/sync-reference-files.R
 ```
 
+## Additional repository contents
 
-## Sample info
+### Sample info
 
-The `sample-info` folder contains metadata files used to create figures and tables.
+The `sample-info/` folder contains metadata files used to create figures and tables.
 
-1. `diagnosis-groupings.tsv`: This tsv file contains one row per `submitted_diagnosis` associated with samples on the ScPCA Portal.
+* `brain-classifications-no-multiplexed.tsv`: This tsv file classifies brain-related diagnoses in the ScPCA Portal into "High-grade glioma" and "Low-grade glioma" for plotting.
+* `celltype-reference-metadata.tsv`: This tsv file contains information about references used for CellAssign cell type annotation on the ScPCA Portal.
```

Comment:
I feel like with the addition of `scpca-project-celltype-metadata.tsv` we don't use this file anymore so we should get rid of it?

Reply:
It's still used here: https://github.com/AlexsLemonade/scpca-paper-figures/blob/ebb7c3ddcbdbb61a94ca965b3e544bcb61880adc/scripts/TableS2_cellassign-ref-summary.R#L13-L15

This file has overall descriptions of the references, but the other one is only names & project. We could combine the files into one probably, but I don't think that's necessary. Code works, good enough!